### PR TITLE
Portal Home Page  Implementation

### PR DESCRIPTION
**Pull request by willy1989cv**
Issue: #4 

**Changes**

- Header
- Footer
- Hero with Video: missing video file
- Partners showcase
- Recenlty added Datasets section
- RWD

### CODE REVIEW

File: frontend/src/components/home/mainSection/DatasetsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        </p>
+      </div>
+
+      <div className="mt-16 grid grid-cols-1 gap-[65px] md:grid-cols-2 lg:grid-cols-3">
+        {_datasets.map((dataset, i) => {
+          return (
+            <div key={`recent-${i}`}>
+              <div className="dataset-card flex flex-col gap-4">
+                {/*Badge*/}
+                <span className=" flex w-fit gap-1 rounded-[6px] bg-[#FDF6B2] px-[10px] py-[2px] text-xs font-medium text-[#723B13]">
```
Comment (luccasmmg) [inline frontend/src/components/home/mainSection/DatasetsSection.tsx:67]: These same "badges" are used in a number of other pages

- run `npx shadcn-ui@latest add badge`
- Change the tailwind code in `components/ui/badge` to match this https://tailwindui.com/components/application-ui/elements/badges#component-78b26cb356fb02423d5da90b0e8fe645
- There should be a prop to provide 1 - An icon 2 - If its on the left or righthand side

File: frontend/src/components/home/mainSection/DatasetsSection.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      <div className="mx-auto text-center lg:max-w-[640px]">
+        <Heading>Recently added</Heading>
+        <p className="mt-4 text-xl font-normal text-gray-500">
+          Explore available datasets and gain valuable insights into the
+          transportation trends globally
+        </p>
+      </div>
+
+      <div className="mt-16 grid grid-cols-1 gap-[65px] md:grid-cols-2 lg:grid-cols-3">
+        {_datasets.map((dataset, i) => {
```
Comment (luccasmmg) [inline frontend/src/components/home/mainSection/DatasetsSection.tsx:62]: There is a small separator missing i believe
image

File: frontend/src/components/home/heroSection/Hero.tsx [hunk 3]
```diff
@@ -1,62 +1,45 @@
...
-                />
-              </div>
-            </div>
+    <section className="pb-[96px] pt-[64px]">
+      <div className="container">
+        <div className="flex items-center gap-x-16 xl:items-start">
+          <div className="w-full">
+            <Link
+              href="#"
+              className="flex w-fit items-center rounded-[14px] bg-[#F3F4F6] p-2"
```
Comment (luccasmmg) [inline frontend/src/components/home/heroSection/Hero.tsx:16]: I think changing this p-2 to p-1.5 makes it look more like the mockup
image

**Review by luccasmmg — CHANGES_REQUESTED** [review]

File: frontend/src/components/home/mainSection/DatasetsSection.tsx (hunk 1 above)
Comment (willy1989cv) [reply frontend/src/components/home/mainSection/DatasetsSection.tsx]: fixed

File: frontend/src/components/home/heroSection/Hero.tsx (hunk 3 above)
Comment (willy1989cv) [reply frontend/src/components/home/heroSection/Hero.tsx]: fixed

**Pull request merged**